server/service/osquery_utils/queries.go

```diff
@@ -129,23 +129,17 @@ var hostDetailQueries = map[string]DetailQuery{
 		},
 	},
 	"os_version": {
```

Comment:
should we change this name since this isn't used to get the version anymore?

Reply:
Definitely open to alternative names :) I thought about changing it to "platform" or something along those lines but didn't have any great ideas. Then I noticed that the detail query names tended to track the osquery table name so I left it alone for now.

Reply:
> Then I noticed that the detail query names tended to track the osquery table name so I left it alone for now.

+1.